WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php

```diff
@@ -111,6 +112,28 @@ public function register() {
 		];
 	}
 
+	/**
+	 * Reset short echo context tracking variable for a new file.
+	 *
+	 * @since 2.3.4
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+	 * @param int                         $stackPtr  The position of the current token
+	 *                                               in the stack passed in $tokens.
+	 *
+	 * @return int|void Integer stack pointer to skip forward or void to continue
+	 *                  normal file processing.
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+		static $current_file;
```

Comment:
I'd suggest assigning the `$current_file` to a `private` property in the sniff (with a default of an empty string), rather than using a `static` variable.

Reply:
For what reason?

Reply:
Handled in [`f1cd93c` (#748)](https://github.com/Automattic/VIP-Coding-Standards/pull/748/commits/f1cd93c77c7a3dcb70348897f2bcce835e161763).

Reply:
Reason: consistency (the `$in_short_echo` tracker is also a property) and reducing "magic".